da/go.mod

```diff
@@ -7,38 +7,169 @@ replace github.com/rollkit/rollkit/core => ../core
 replace github.com/rollkit/rollkit => ../
 
 require (
-	cosmossdk.io/log v1.5.0
+	cosmossdk.io/log v1.5.1
 	github.com/filecoin-project/go-jsonrpc v0.7.1
-	github.com/rollkit/rollkit/core v0.0.0-00010101000000-000000000000
+	github.com/rollkit/rollkit v0.0.0-00010101000000-000000000000
```

Comment:
Is the addition of the rollkit dependency in da expected?

Reply:
Yeah, this looks like a potential source of problems.
Mocks are could be in `core` or need separate `go.mod` :thinking:

Reply:
i can generate a go.mod for it. was thinking of doing the same. At least here we get top down, release core, then rollkit then da and others instead of inverse. for rollkit and da.

Reply:
rollkit will still be a indirect dep even by making mocks a go.mod. Ill try something else

Reply:
created an internal mocks directory for da to avoid the import as even the mocks depend on main rollkit